src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1

```diff
@@ -0,0 +1,109 @@
+﻿#  
+# Module manifest for module 'Microsoft.Azure.Commands.Insights'  
+#  
+# Generated by: Microsoft Corporation  
+#  
+# Generated on: 9/19/2015
+#  
+  
+@{  
+  
+# Version number of this module.  
+ModuleVersion = '0.15.4' 
+  
+# ID used to uniquely identify this module  
+GUID = 'DA5816B5-97A6-4301-9AA0-72CC68C79F20'  
+  
+# Author of this module  
+Author = 'Microsoft Corporation'  
+  
+# Company or vendor of this module  
+CompanyName = 'Microsoft Corporation'  
+  
+# Copyright statement for this module  
+Copyright = 'Microsoft Corporation. All rights reserved.'    
+  
+# Description of the functionality provided by this module  
+Description = 'Microsoft Azure PowerShell - Storage management cmdlets for Azure Stack'  
+  
+# Minimum version of the Windows PowerShell engine required by this module  
+PowerShellVersion = '3.0'  
+  
+# Name of the Windows PowerShell host required by this module  
+PowerShellHostName = ''  
+  
+# Minimum version of the Windows PowerShell host required by this module  
+PowerShellHostVersion = ''  
+  
+# Minimum version of the .NET Framework required by this module  
+DotNetFrameworkVersion = '4.5.2'  
+  
+# Minimum version of the common language runtime (CLR) required by this module  
+CLRVersion='4.0'  
+  
+# Processor architecture (None, X86, Amd64, IA64) required by this module  
+ProcessorArchitecture = 'None'  
+  
+# Modules that must be imported into the global environment prior to importing this module  
+RequiredModules = @( @{ ModuleName = 'AzureRM.Profile'; ModuleVersion = '3.3.1'})
```

Comment:
Update to 3.4.0

Reply:
This folder was supposed to be gone.  I will go back and delete this folder.  Not sure what happened.